examples/tests/fail.earth

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.11
+test:
+    # intentionally cause a failure (this test is to ensure failures are caught)
+    RUN false
```

Comment:
Wondering if we can also put some output in here (eg `RUN echo "expected output"; false`) and then check that the output is printed (`.... +test | grep "expected output"`).

Reply:
good plan, I changed it to a `ls /this-will-fail` then I can grep for that fake path shows up in the error message.